src/nikhilMJun20/FindItemId.java

```diff
@@ -0,0 +1,47 @@
+/*Assignment 33 : 22nd July 2020
+
+Extract ItemId from below input
+
+String input = https://www.facebook.com/technocredits/itemId=12345/
+String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+String input = https://www.facebook.com/technocredits/empid=123/
+
+Output : 
+item id is 12345 
+        OR
+no item id present.
+*/
+
+package nikhilMJun20;
+
+public class FindItemId {
+	void displayItemId(String str, String toBeFound) {
+		String itemId = "";
+		for(int index = 0; index < str.length()-toBeFound.length(); index++) {
+			if((str.substring(index, (index+toBeFound.length()))).equals(toBeFound)) {
```

Comment:
--No issues in the logic
-- However just a suggestion you can try this:
 int firstindex = str.indexOf("itemId");
String temp = str.substring(firstindex); 
and then proceed further it will be a little less complex :)

Reply:
@VaishnaviVJun20:
Yes, I fully agree. I will implement the suggestion.
Thanks!